control-plane/agents/src/common/errors.rs

```diff
@@ -934,6 +936,12 @@ impl From<SvcError> for ReplyError {
                 source,
                 extra,
             },
+            SvcError::VolWouldExceedCapacity {} => ReplyError {
+                kind: ReplyErrorKind::OutOfRange,
```

Comment:
I was in two minds about this (see the question in the other PR).
I could change it to ResourceExhausted, but wondered if it would be better to separate this condition (an arbitrary limit) than some other error condition where there is a physical lack of storage.
Happy to make the change, depending what others think.

Reply:
I think we can add a new variant to `ReplyErrorKind`, `CapacityLimitExceeded` ?

Reply:
> I think we can add a new variant to `ReplyErrorKind`, `CapacityLimitExceeded` ?

Yes that would be good.

Reply:
Fixed